tasks/jscs.js

```diff
@@ -13,12 +13,18 @@ module.exports = function( grunt ) {
 
                 // null is a default value, but its equivalent to `true`,
                 // with this way it's easy to distinguish specified value
-                config: null
+                config: null,
+
+                fix: false
             } ),
 
             jscs = new JSCS( options ),
             checks = this.filesSrc.map( function( path ) {
```

Comment:
Looks good, although, we could do a small optimization here -

```js
this.filesSrc.map( jscs.getOptions().fix ? jscs.fix.bind( jscs ) : jscs.check.bind( jscs ) )
```
Or some variation of it, that way, that `if` would not be called on every iteration.